game.py

```diff
@@ -0,0 +1,252 @@
+"""
+Created on Thursday Oct 15, 2017
+SkyRoads: MiniProject 4
+
+@authors: Felix Eberhardt & Shreya Rangarajan
+"""
+from threading import Timer
+import time
+import random
+import os, sys, time
+import pygame
+from pygame.locals import *
+
+if not pygame.font:
+    print('Warning, fonts disabled')
+
+running = True
+speed = 3
+
+"""
+Define all the inital variables
+"""
+white = (255, 255, 255)
+black = (0, 0, 0)
+green = (0, 200, 0)
+red = (255, 0, 0)
+green2 = (0, 255, 0)
+blue = (0, 0, 255)
+brown = (165, 42, 42)
+
+car = "audi_1.png"
+barrier = "concrete.png"
+road = "road.png"
+barrier2 = "concrete.png"
+
+display_width = 1024
+display_height = 750
+
+background_size = (display_width, display_height)
+car_size = (100, 67)
+
+"""
+Initialize the game
+"""
+pygame.init()
+myfont = pygame.font.SysFont("monospace", 40)
+
+screen = pygame.display.set_mode(background_size)
+clock = pygame.time.Clock()
+timer = pygame.time.get_ticks()
+barrier_limit = random.randint(1,3)
+
+background_image = pygame.image.load(road).convert()
+player_image = pygame.image.load(car).convert()
+player_image_rect = player_image.get_rect()
+barriers_list = []
+barriers_list_rect = []
+barriers_list_pos = []
+concrete_img = pygame.image.load(barrier)
+barriers_list.append(concrete_img)
+barriers_list_pos.append([420,100])
+concrete_img_rect = concrete_img.get_rect()
+barriers_list_rect.append(concrete_img_rect)
+

```

Comment:
Ah, I think you left this code at top level instead of inside a function because most of what it's doing is initializing global variables.

Consider making this attributes of a class, e.g. `Game`.